components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.endpoint/src/main/resources/org.wso2.carbon.identity.configuration.management.yaml

```diff
@@ -314,25 +314,25 @@ paths:
         - Resource
     put:
       description: |
-        This API is used to store or replace the resource given by the user.
+        This API is used to create or replace a resource. 
       x-wso2-request: |
         curl -k -X PUT "https://localhost:9443/api/identity/config-mgt/v1.0/resource/{resource-type}" -H "accept: application/json" -H "Content-Type: application/json" -H "Authorization: Basic <Base64Encoded[username:password]>" -d '{"name": "string","attributes": [{"id": "string","key": "string","value": "string"}]}'
       x-wso2-curl: |
         curl -k -X PUT https://localhost:9443/api/identity/config-mgt/v1.0/resource/e-mail -H "accept: application/json" -H 'Content-Type: application/json' -H 'Authorization: Basic YWRtaW46YWRtaW4=' -d '{"name": "smtp","attributes": [{"key": "from","value": "3abc.com"},{"key": "to","value": "3wso2.com"}]}'
       x-wso2-response: |
-        {"resourceId":"7060b0c6-000b-47ed-86ef-396ff5cb8203","tenantDomain":"carbon.super","resourceName":"smtp","resourceType":"e-mail","lastModified": "2019-01-24T10:15:51Z","created": "2019-01-24T10:15:51Z","attributes":[{"id":null,"key":"from","value":"3abc.com"},{"id":null,"key":"to","value":"3wso2.com"}],"files":[]}
+        {"resourceId":"4e2eb3fc-99d9-4733-96b1-dcbf3d2257a2","tenantDomain":"carbon.super","resourceName":"smtp","resourceType":"e-mail","lastModified":"2019-02-07T11:37:45.815Z","created":"2019-02-07T11:37:45.815Z","attributes":[{"key":"from","value":"3abc.com"},{"key":"to","value":"3wso2.com"}],"files":[]}
       summary: |
-        Add or Replace the resource
+        Create or replace resource
       # put parameters:
       parameters:
         - name: resource-type
           in: path
-          description:  This represents the type of the resource to be added and can either be the name or id.
+          description:  This represents the type of the resource that is to be added/replaced and can either be the resource-type name or ID.
```

Comment:
This value can only be the resource name. Shall we change it accordingly?

Reply:
Fixed with c2af0a29f9313baf5105a4898a9e26baceae00fa